src/catkin_pkg/cmake.py

```diff
@@ -38,17 +38,17 @@
 
 
 def get_metapackage_cmake_template_path():
-    '''
-    Returns the location of the metapackage CMakeLists.txt CMake template.
+    """
+    Return the location of the metapackage CMakeLists.txt CMake template.
 
     :returns: ``str`` location of the metapackage CMakeLists.txt CMake template
-    '''
+    """
     rel_path = os.path.join('templates', 'metapackage.cmake.in')
     return os.path.join(os.path.dirname(__file__), rel_path)
 
 
-def configure_file(template_file, environment):
-    '''
+def configure_file(template_file, environment):  # noqa: ignore=D402
```

Comment:
The reason why this comment suppresses all warnings is the way it is written. Please use `# noqa: D402` to only suppress that specific warning. That style ensures that other warnings are not suppressed.

Reply:
Fixed.  

There are some other incorrectly suppressed warnings in this project, I will address them in a separate PR.